lib/redshift_serverless_namespace_stack.py

```diff
@@ -14,15 +16,17 @@ def __init__(self, scope: cdk.Construct, construct_id: str, **kwargs) -> None:
         redshift_full_command_access = iam.ManagedPolicy.from_aws_managed_policy_name(
             "AmazonRedshiftAllCommandsFullAccess")
 
-        role = iam.Role(self, "sls-test-role",
+        role = iam.Role(self, "LMD20RedshiftServerlessRole",
                         assumed_by=iam.ServicePrincipal("redshift.amazonaws.com"),
                         managed_policies=[redshift_full_command_access])
 
+        secret = secretsmanager.Secret(self, "LMD20RedshiftPassword")
```

Comment:
This isn't really bad but I will have to update this code and move this secret to the file that's responsible for pushing it to AWS and keeping the alias.